models/articles.model.js

```diff
@@ -13,4 +13,11 @@ exports.fetchArticleByID = (article_id) => {
             }
         return article
         });
+}
+
+exports.fetchAllArticles = () => {
+    let query = 'SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;'
```

Comment:
Brilliant SQL query - use of the `COUNT` aggregate function and `LEFT JOIN` will make this work a treat, well done!